samples: wifi: twt: Fix build with default configuration

D-Triveni opened the pull request:
TWT sample fails without extra `TRAFFIC_GEN_REMOTE_IPV4_ADDR` config, so added it as a default configuration in prj.conf.

NordicBuilder commented:
## Test specification

### [CI/Jenkins/NRF](https://jenkins-ncs.nordicsemi.no/job/latest/job/sdk-nrf/job/PR-16157/1/)
- Skipped

### [CI/Jenkins/integration](https://jenkins-ncs.nordicsemi.no/job/latest/job/sdk-nrf/job/PR-16157/1/)
- Skipped



*Note: This message is automatically posted and updated by the CI*

krish2718 commented on samples/wifi/twt/prj.conf:
```diff
@@ -90,5 +90,7 @@ CONFIG_NET_CONFIG_MY_IPV4_GW="192.168.1.1"
 
 CONFIG_TRAFFIC_GEN=y
+# IPv4 address of the host running the traffic generator server
+CONFIG_TRAFFIC_GEN_REMOTE_IPV4_ADDR="192.168.0.1"
```
This was intentionally left out of default config, we don't have put some default address without user's knowledge, it will start failing and user has no idea why. So, a build assert is added to force the user to choose an IP, see https://github.com/nrfconnect/sdk-nrf/blob/main/samples/wifi/twt/modules/traffic_gen/src/main.c#L319

D-Triveni commented:
> Then in the sample documentation we can stress a bit better that this Kconfig symbol needs to be looked-at and modified accordingly (if needed) and state the reason for that (e.g. otherwise the sample can start to fail).

Addressed.

NordicBuilder commented:
You can find the documentation preview for this PR at this <a href='http://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-16157
'>link</a>. It will be updated about 10 minutes after the documentation build succeeds.

*Note: This comment is automatically posted by the Documentation Publishing GitHub Action.*

krish2718 commented on samples/wifi/twt/README.rst:
```diff
@@ -95,6 +95,10 @@ The following is an example of the CLI command:
 
    west build -b nrf7002dk/nrf5340/cpuapp
 
+.. note::
```
```suggestion
.. important::
```

ioannisg reviewed (approved):
lgtm

richabp commented on samples/wifi/twt/README.rst:
```diff
@@ -95,6 +95,10 @@ The following is an example of the CLI command:
 
+.. important::
+
+   It is mandatory to set the :kconfig:option:`CONFIG_TRAFFIC_GEN_REMOTE_IPV4_ADDR` Kconfig option to the IPv4 address of the host running the traffic generator server. If this is set incorrectly, the sample may fail to work as expected.
```
I see that this PR has been merged, thus the changes can be implemented in another PR.
The sentence in line 100 should begin on a new line, following the one sentence per line syntax rule in RST.
